### DISCUSSION TITLE
Questions about settings screen

### DISCUSSION THREAD

**Discussion by evdgriek** [creepymonster/GlucoseDirect#456] [General]
Hi,
I'm trying to find out how to set-up de app on my iPhone. 
In general is see data on my lock screen and Apple Watch, top it's great. But I like to know what to do with:
Glucose live activity
Glucose reading
Both are switched off, but what do they do

Like to hear from you
Keep on the good work

**Comment by creepymonster** [comment]
Hello,

Live Activities are a great new feature of iOS 16.

To enable it, go to iOS Settings > Glucose Direct > Live Activities.

Then activate it in Glucose Direct under Settings.

From then on, the lock screen will show glucose readings in “real time”.

Many greetings
Reimar

**Reply by rahuldhev** [reply]
Does live activities work for the Freestyle Libre 3?
